packages/transaction-controller/src/gas-flows/OptimismLayer1GasFeeFlow.test.ts

```diff
@@ -82,26 +69,33 @@ describe('OptimismLayer1GasFeeFlow', () => {
       const flow = new OptimismLayer1GasFeeFlow();
       const response = await flow.getLayer1Fee(request);
 
-      expect(buildUnserializedTransactionMock).toHaveBeenCalledWith(
```

Comment:
I was thinking to assert it in the tests, but `serializedTransaction` is a Buffer. Then I leave this test with only asserting `contractGetL1FeeMock` to be called once. 

We can still create an expected Buffer in the tests but I see no value since it's unreadable.

Reply:
I initially thought asserting this but then I give up and remove it because `serializedTransaction` is a buffer. And asserting a buffer in tests were not readable at all. 

On this iteration I added an assertion that we buildup that `serializedTransaction` from exact `txParams` spying on `TransactionFactory.fromTxData`, hope that it make sense now.

Reply:
We could theoretically mock the serialize to return a value we could assert, plus we could check that the call to serialize was what we expect also, but it's all diminishing returns the more complex it gets.